doc/admin/troubleshooting.md

```diff
@@ -289,6 +289,37 @@ To remove the shared memory segment left behind by I/O Engine instance 1, issue:
 sudo ipcrm -M 0x10242049
 ```
 
+### Server Start Issues
+1. Read the log located in the "control_log_file:"
+1. Verify that the daos_server process is not currently running.
```

Comment:
So presume all the prefixes get automatically updated to incremental integers rather than printing as all 1s?